src/kernels/composable_kernel/include/utility/amd_inline_asm.hpp

```diff
@@ -9,18 +9,37 @@ namespace ck {
 // outer-product: c[i,j] += inner_product(a[i], b[j])
 __device__ void amd_assembly_outer_product_1x2(float a, float b0, float b1, float& c0, float& c1)
 {
+#if CK_USE_AMD_V_FMAC_F32
```

Comment:
Why not removing v_mac_f32 completely?

Reply:
@asroy 
- gfx900 foes not have FMAC_F32
- MAC_F32 -> FMAC_F32 may lead to performance drop on gfx9. More info upon request.